Add empty mechanical concept - (Task #155)

franzTobiasDLR opened the pull request:
Closes #155

franzTobiasDLR committed:
Add concept projects for an empty mechanical project - (Task #155)

SaMuellerDLR commented on de.dlr.sc.virsat.model.extension.mechanical/concept/concept.concept:
```diff
@@ -0,0 +1,6 @@
+Concept de.dlr.sc.virsat.model.extension.mechanical description "Concept for mechanical engineering" version 0.1
```
Style thingy. it should be `Concept de.dlr.sc.virsat.model.extension.mechanical description "Concept for mechanical engineering" version 0.1 {` with the `{` on the same line.

SaMuellerDLR commented on de.dlr.sc.virsat.model.extension.mechanical/concept/concept.concept:
```diff
@@ -0,0 +1,6 @@
+Concept de.dlr.sc.virsat.model.extension.mechanical description "Concept for mechanical engineering" version 0.1
+{
```
Also the concept should have the header 

`/*******************************************************************************
 * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License 2.0 which is available at
 * http://www.eclipse.org/legal/epl-2.0.
 *
 * SPDX-License-Identifier: EPL-2.0
*******************************************************************************/`

like the other concepts.

franzTobiasDLR committed:
Add copyright header to concept and fix style - (Task #155)

franzTobiasDLR committed:
Remove empty src folders from build path - (Task #155)

SaMuellerDLR reviewed (dismissed):
Approved!

Commit:
Cleaned up test cases to improve coverage - (Task #155)

- Added new test case for SeiValidator.
- Adjusted existing and generated test cases.
- Removed migrator for old v1 concept.

---
Task #155: Add empty Mechanical Engineering concept

SaMuellerDLR reviewed (approved):
Approved!
